src/components/Dasboard/Cards/cardHooks/GetCardType.ts

```diff
@@ -152,7 +152,10 @@ export function GetCardTypeAndValues(props: CardProps) {
 			pieCardConfig.loading = loading;
 			fetchedData = data;
 		}
-
+		console.log("fetched", fetchedData);
+		if (fetchedData === 0) {
```

Comment:
`fectData === 0`. What does it mean?

Reply:
we get 0 value from backend if the list is empty .. it should be an empty array

Reply:
in GetBudgetCategory